pkgs/applications/audio/helio-workstation/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub
+, alsaLib, freetype, xorg, curl, libGL, libjack2, gnome3
+, pkgconfig, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "helio-workstation";
+  version = "2.2";
+
+  src = fetchFromGitHub {
+    owner = "helio-fm";
+    repo = "helio-workstation";
+    rev = "${version}";
+    fetchSubmodules = true;
+    sha256 = "16iwj4mjs1nm8dlk70q97svp3vkcgs7hdj9hfda9h67acn4a8vvk";
+  };
+
+  buildInputs = [
+    alsaLib freetype xorg.libX11 xorg.libXext xorg.libXinerama xorg.libXrandr
+    xorg.libXcursor xorg.libXcomposite curl libGL libjack2 gnome3.zenity
+  ];
+
+  nativeBuildInputs = [ pkgconfig makeWrapper ];
+
+  preBuild = "cd Projects/LinuxMakefile";
+  buildFlags = [ "CONFIG=Release64" ];
```

Comment:
Could you make this work for aarch64-linux too?

Reply:
Well, I don't really know how to do this.